docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/PriorTrainerSample.cs

```diff
@@ -63,7 +63,7 @@ public static void Example()
             //  Positive Precision: 0.50
             //  Positive Recall: 1.00
             //  LogLoss: 1.05
-            //  LogLossReduction: -4.89
```

Comment:
Why does it change the baseline?

Reply:
It didn't the documentation was stable. I verified this without the prior trainer change from the master branch (removed the change that changes label type to boolean) and below is the output:

Accuracy: 0.50
AUC: 0.50
F1 Score: 0.67
Negative Precision: 0.00
Negative Recall: 0.00
Positive Precision: 0.50
Positive Recall: 1.00
Log Loss: 1.05
Log Loss Reduction: -0.05
Entropy: 1.00